app/src/main/res/layout/view_facilitypicker.xml

```diff
@@ -33,7 +33,7 @@
             android:id="@+id/searchEditText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:hint="@string/facilitypicker_search_hint"
+            android:hint="@string/facilitypicker_search_hint_new"
```

Comment:
We don't need to change the key to this string. Just the value for it. Can you undo this?